Sources/Gtk/Utility/CSS/CSSProperty.swift

```diff
@@ -0,0 +1,47 @@
+/// A type safe CSS property
+///
+/// All CSS properties supported in Gtk: https://docs.gtk.org/gtk4/css-properties.html
```

Comment:
Something like this will show up nicer in the documentation site made with DocC:
```swift
/// For a list of all supported CSS properties, see [Gtk's CSS documentation](https://docs.gtk.org/gtk4/css-properties.html).
```